test/src/com/xilinx/rapidwright/edif/TestEDIFNetlist.java

```diff
@@ -450,6 +450,38 @@ public void testBussedPortNamingCollision(@TempDir Path path) {
         Assertions.assertEquals(port1.getWidth(), testPort1.getWidth());
     }
 
+    @Test
+    public void testBussedPortReNamingCollision(@TempDir Path path) {
+        final EDIFNetlist origNetlist = EDIFTools.createNewNetlist("test");
+
+        EDIFCell top = origNetlist.getTopCell();
+
+        EDIFCellInst ff = top.createChildCellInst("ff", Design.getPrimitivesLibrary().getCell("FDRE"));
+        origNetlist.getHDIPrimitivesLibrary().addCell(ff.getCellType());
+
+        String portName = "unfortunate_name";
+
+        // Create two bussed ports with a common root name, one having a trailing '_' underscore
+        EDIFPort port0 = top.createPort(portName + "[2:0]", EDIFDirection.INOUT, 3);
+        EDIFPort port1 = top.createPort(portName + "_[1:0]", EDIFDirection.INOUT, 2);
+
+        EDIFNet net0 = top.createNet("net0");
+        net0.createPortInst(port0, 0);
+        net0.createPortInst("D", ff);
+
+        EDIFNet net1 = top.createNet("net1");
+        net1.createPortInst(port1, 1);
+        net1.createPortInst("R", ff);
```

Comment:
Just to be extra pedantic (it seems to work!)

```suggestion
        // Create two bussed ports with a common root name, one having a trailing '_' underscore
        EDIFPort port0 = top.createPort(portName + "[2:0]", EDIFDirection.INOUT, 3);
        EDIFPort port1 = top.createPort(portName + "_[1:0]", EDIFDirection.INOUT, 2);
        // And for good measure, throw in a single-bit port too with the same root
        EDIFPort port2 = top.createPort(portName, EDIFDirection.INOUT, 1);

        EDIFNet net0 = top.createNet("net0");
        net0.createPortInst(port0, 0);
        net0.createPortInst("D", ff);

        EDIFNet net1 = top.createNet("net1");
        net1.createPortInst(port1, 1);
        net1.createPortInst("R", ff);
        
        DIFNet net2 = top.createNet("net2");
        net2.createPortInst(port2);
        net2.createPortInst("Q", ff);
```

This is the relevant part of the resulting EDIF:
```
  (Library work
    (edifLevel 0)
    (technology (numberDefinition ))
   (cell test (celltype GENERIC)
     (view netlist (viewtype NETLIST)
       (interface  
        (port (array (rename unfortunate_name_BUS_ "unfortunate_name[2:0]") 3) (direction INOUT))
        (port unfortunate_name (direction INOUT))
        (port (array (rename unfortunate_name_ "unfortunate_name_[1:0]") 2) (direction INOUT))
       )
       (contents
         (instance ff (viewref netlist (cellref FDRE (libraryref hdi_primitives))))
         (net net0 (joined
          (portref D (instanceref ff))
          (portref (member unfortunate_name_BUS_ 0))
          )
         )
         (net net2 (joined
          (portref Q (instanceref ff))
          (portref unfortunate_name)
          )
         )
         (net net1 (joined
          (portref R (instanceref ff))
          (portref (member unfortunate_name_ 1))
          )
         )
       )
     )
   )
  )
```

I think that looks okay -- it even gets accepted by Vivado:
![image](https://github.com/user-attachments/assets/1263f310-5e88-4c69-b348-b8af3148d03a)

Reply:
And just to satisfy my curiosity, here's what Vivado itself outputs:
```
       (interface
        (port unfortunate_name (direction INOUT))
        (port (array (rename unfortunate_name_HDI_0 "unfortunate_name[2:0]") 3) (direction INOUT))
        (port (array (rename unfortunate_name_ "unfortunate_name_[1:0]") 2) (direction INOUT))
       )
```